data/projects.js

```diff
@@ -81,5 +81,27 @@ coala should support generating metrics for your code.",
 		"initiatives" : ["GSoC"],
 		"tags" : ["Core", "Bears"],
 		"markdown": "implement_metrics_for_coala.md"
+	},
+	{
+		"name" : "Optimize Caching",
+		"desc" : "Improve coala's performance by implementing caching strategies.",
+		"requirements" : [
+			"At least one patch to the coala core should be accepted and merged.",
+			"The student should be familiar with ongoing technical cEPs.",
+			"The student must have proved his or her throughout understanding of the coala core."
```

Comment:
"The proposal needs to show some familiarity with coala's caching implementation"

or your typical

"The proposal needs to contain a clear concept." ?